simbad/rotsearch/amore_search.py

```diff
@@ -60,298 +57,25 @@ class AmoreRotationSearch(object):
-                ])
-                stdin = stdin.format(shres, intrad, pklim, npic, rotastep)
-        else:
-            msg = "Incorrect combination of input arguments"
-            logger.critical(msg)
-            raise RuntimeError(msg)
-
-        return cmd, stdin
-
-    @staticmethod
-    def submit_chunk(chunk_scripts, run_dir, nproc, job_name, submit_qtype, submit_queue, monitor):
-        """Submit jobs in small chunks to avoid using too much disk space
-        
-        Parameters
-        ----------
-        chunk_scripts : list
-            List of scripts for each chunk
-        nproc : int, optional
-            The number of processors to run the job on
-        job_name : str
-            The name of the job to submit
-        submit_qtype : str
-            The cluster submission queue type - currently support SGE and LSF
-        submit_queue : str
-            The queue to submit to on the cluster
-
-        """
-        j = Job(submit_qtype)
-        j.submit(chunk_scripts, directory=run_dir, name=job_name,
-                 nproc=nproc, queue=submit_queue, permit_nonzero=True)
-        interval = int(numpy.log(len(chunk_scripts)) / 3)
-        interval_in_seconds = interval if interval >= 5 else 5
-        j.wait(interval=interval_in_seconds, monitor=monitor)
-
-    @staticmethod
-    def tabfun(amore_exe, xyzin1, xyzout1, table1, x=200, y=200, z=200, a=90, b=90, c=120):
-        """Function to perform amore table function
-
-        Parameters
-        ----------
-        amore_exe : str
-           The path to the AMORE executable
-        xyzin1 : str
-           Path to input model
-        xyzout1 : str
-           Path to the output model
-        table1 : str
-           Path to the output table file
-        x : int, float, optional
-           x value for minimal box [default: 200]
-        y : int, float, optional
-           y value for minimal box [default: 200]
-        z : int, float, optional
-           z value for minimal box [default: 200]
-        a : int, float, optional
-           alpha value for minimal box [default: 90]
-        b : int, float, optional
-           beta value for minimal box [default: 90]
-        c : int, float, optional
-           gamma value for minimal box [default: 120]
-
-        Returns
-        -------
-        file
-            Output PDB for use in rotfun
-        file
-            Output table file for use in rotfun
+        self._search_results = None
 
-        """
-        cmd = [amore_exe, 'xyzin1', xyzin1,
-               'xyzout1', xyzout1, 'table1', table1]
-        stdin = os.linesep.join([
-            "TITLE: Produce table for MODEL FRAGMENT",
-            "TABFUN",
-            "CRYSTAL {0} {1} {2} {3} {4} {5} ORTH 1",
-            "MODEL 1 BTARGET 23.5",
-            "SAMPLE 1 RESO 2.5 SHANN 2.5 SCALE 4.0",
-        ])
-        stdin = stdin.format(x, y, z, a, b, c)
-        return cmd, stdin
+    def run_pdb(self, *args, **kwargs):
```

Comment:
My mistake above, this is still here. Do we need this?

Reply:
I left it in to not break it, but probs going to remove it to address this and previous comment

Reply:
From looking at comand_line/__init__, every where it was used has been replaced by run and seeing as we've not release this yet we don't have to worry about continuity just yet.

Reply:
Addressed in c01b3ab